README.md

```diff
@@ -316,8 +320,58 @@ The Realm Dart package is `realm_dart`
 
     _*This file should be committed to source control_
 
-* For more usage of Realm Dart see the Realm Flutter usage above.
+* For more information on using the Realm Dart Standalone SDK see the Realm Flutter SDK usage above.
+
+# Device Sync with Realm Flutter and Dart Standalone SDKs
```

Comment:
```suggestion
# Device Sync with Realm Flutter and Dart SDKs 
```
The Dart Standalone goes with many names so we could skip mentioning it too much. Not sure which one is the most popular or if there is one  so I suggest we just drop it here.

Reply:
Replaced with "Sync data with Realm Flutter and Dart using Device Sync"